_includes/pyos-blockquote.html

```diff
@@ -7,5 +7,6 @@
     {% if include.author %}<strong>~{{ include.author }}{% endif %}</strong>{%
     if include.event %}, <em>{{ include.event }}</em>{% endif %}
   </footer>
+  <div class="end"></div>
```

Comment:
We are using an `:after` element for the quotes. this adds a new div below the footer (which is the quote author) that after is applied to. This makes it easier to align the close quote with the actual text.